### ISSUE TITLE
API error: Connection refused error in logs

### ISSUE THREAD

**Issue by danieldotweinert** [hjdhjd/homebridge-unifi-protect#1138]
### Homebridge UniFi Protect Version

7.8.2

### Homebridge Platform and OS

macOS Monterey

### Homebridge Version

1.8.4

### Node Version

v20.18.0

### UniFi OS Version

v4.0.18

### UniFi Protect Controller Version

v5.0.45

### Describe the problem

Currently getting an error in the Homebridge logs that says `API error: Connection refused.` It doesn't appear to effect the cameras in HomeKit I am just wondering if something else is wrong and I don't want to get throttled from too many API calls. Have you seen this before?

### Homebridge HBUP JSON configuration

```json
{
    "controllers": [
        {
            "address": "192.168.10.151",
            "password": "############",
            "username": "HomeBridge"
        }
    ],
    "name": "UniFi Protect",
    "options": [
        "Disable.Device.F4E2C60DB079",
        "Enable.Video.Transcode.Hardware.F4E2C60DD135",
        "Enable.Video.Transcode.Hardware.F4E2C60DD303",
        "Enable.Video.Transcode.Hardware.F4E2C60E1B8A",
        "Enable.Video.Transcode.Hardware.F4E2C60D63A8",
        "Enable.Video.Transcode.Hardware.F4E2C6731A83",
        "Enable.Device.StatusLed.F4E2C60DD135",
        "Enable.Device.SyncName.F4E2C60DD135",
        "Enable.Device.StatusLed.F4E2C6731A83",
        "Enable.Device.SyncName.F4E2C6731A83",
        "Enable.Device.StatusLed.F4E2C60DD303",
        "Enable.Device.SyncName.F4E2C60DD303",
        "Enable.Device.StatusLed.F4E2C60E1B8A",
        "Enable.Device.SyncName.F4E2C60E1B8A",
        "Enable.Device.StatusLed.F4E2C60D63A8",
        "Enable.Device.SyncName.F4E2C60D63A8",
        "Enable.Device.StatusLed.F4E2C60FF9D1",
        "Enable.Device.SyncName.F4E2C60FF9D1",
        "Enable.Device.SyncName.F4E2C60DB079",
        "Enable.Video.Transcode.Hardware.F4E2C60FF9D1",
        "Enable.Device.SyncName.F4E2C60D5CFF",
        "Enable.Device.StatusLed.F4E2C60D5CFF",
        "Enable.Video.Transcode.Hardware.F4E2C60D5CFF"
    ],
    "_bridge": {
        "username": "0E:29:B2:02:00:A6",
        "port": 30881
    },
    "platform": "UniFi Protect"
```

### Relevant log output

```shell
API error: Connection refused.
```

### Acknowledgment that you are only running UniFi OS and UniFi Protect releases from the Ubiquiti Official release channel

- [X] I acknowledge that I am not running any early access / beta firmwares or operating systems from either Ubiquiti or Apple.

**Comment by hjdhjd** [comment]
Thanks for the report and no, it doesn’t mean it’s getting too many API calls - HBUP doesn’t spam the controller. The Protect controller firmware has bugs and occasionally doesn’t respond. It’s not an HBUP thing. It happens…ignore it, life will go on. Generally after a few releases after a major version bump Ubiquiti tends to stabilize their firmware quality…given that v5 was pretty recently released, I’d expect it’ll continue to improve.

TL;DR: nothing to see here. Not a bug. Not an issue in HBUP. Protect controller firmware quirkiness/bugs. HBUP will recover just fine when the controller returns to life.
